public/img/gaming.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#424242" d="M40,6c0,0,2,8,2,18c0,5.148-0.53,9.765-1.044,13.035C38,37,24,37,24,37l-4,5H8c0,0-2-8-2-18S8,6,8,6 H40z"/><path fill="#bdbdbd" d="M41.11 36C40.58 39.69 40 42 40 42H19c0-3.31 2.69-6 6-6H41.11zM20.12 25H19v-1.8c0-.993.754-1.8 1.68-1.8h3.36c.309 0 .56-.269.56-.6 0-.331-.251-.6-.56-.6h-4.48V19h4.48c.926 0 1.68.807 1.68 1.8 0 .993-.754 1.8-1.68 1.8h-3.36c-.309 0-.56.269-.56.6V25zM27.96 25h-2.8v-1.2h2.8c.309 0 .56-.269.56-.6v-2.4c0-.993.754-1.8 1.68-1.8H33v1.2h-2.8c-.309 0-.56.269-.56.6v2.4C29.64 24.193 28.886 25 27.96 25zM38.273 22.6H34.12v-1.2h4.153c.266 0 .509-.164.575-.39.081-.275-.013-.471-.076-.564C38.666 20.29 38.501 20.2 38.32 20.2h-4.2V19h4.2c.534 0 1.041.276 1.356.738.321.47.408 1.066.24 1.635C39.703 22.095 39.027 22.6 38.273 22.6z"/><path fill="#bdbdbd" d="M38.273,25H34.12v-1.2h4.153c0.266,0,0.509-0.164,0.575-0.39c0.081-0.275-0.013-0.471-0.076-0.564 C38.666,22.69,38.501,22.6,38.32,22.6h-4.2v-1.2h4.2c0.534,0,1.041,0.276,1.356,0.738c0.321,0.47,0.408,1.066,0.24,1.635 C39.703,24.495,39.027,25,38.273,25z"/><g><path fill="#757575" d="M35 38A1 1 0 1 0 35 40 1 1 0 1 0 35 38zM28 38A1 1 0 1 0 28 40 1 1 0 1 0 28 38z"/></g></svg>
```

Comment:
Minor thing but the PS3 was released in 2006 so we may want something more trendy for the YP 😅